motoman_hc10_support/launch/robot_interface_streaming_hc10dtp_b00.launch

```diff
@@ -0,0 +1,21 @@
+<!--
+  Manipulator specific version of 'robot_interface_streaming.launch'.
+
+  Defaults provided for HC10DTP IP67 (B10):
```

Comment:
Based on the discussion: should this be `B00` instead?

```suggestion
  Defaults provided for HC10DTP IP67 (B00):
```

?

Reply:
Addressed by merges of cjue commits.